admin/tools/docker/index/container/dev/clientNum/Dockerfile

```diff
@@ -0,0 +1,13 @@
+#
```

Comment:
It made the yaml  scripts easier to write and get working (which was not easy) and an actual instance of the system would not have the client apps, as they are only for testing. It would make sense to eventually trim down the containers so they only contain what they really need.
Currently, the containers all share the same qserv/indexbase:dev. The differences for the individual containers really comes down the entrypoint designations and bash scripts, which is small. There could be significant differences between master and worker containers in the future.